app/assets/javascripts/sequence_show.js

```diff
@@ -1,7 +1,29 @@
-import {addCopy, logToGoogle, triggerDownloadModal} from "./utils.js";
+import {addCopy, logToGoogle, triggerDownloadModal, stringTitleize, numberToPercent} from "./utils.js";
 import {showInfoModal} from "./modal.js";
-
```

Comment:
typo: panel

_[Original comment](https://github.ugent.be/unipept/unipept/pull/625#discussion_r13896) by @bmesuere on Wed Feb 28 2018 at 15:10._